### PR TITLE
Support embedding metadata into docx

### PR DESCRIPTION
**Pull request by matt-hayden**
This is in draft one of the requirements strictly implies a pandoc types introspection not carried into the export method.

https://github.com/innolitics/notion-qms/pull/61 stacks on this.

# Author's Checklist

Before assigning reviewers, step through the following checklist. Check each item once it's completed. If an item is skipped, write an explanation below the item.

- [ ] **Construction:** Review the code diff and confirm there's no dead code, debug code, or unnecessary comments.
- [ ] **Requirements:** Review the requirements in the issue for this PR and confirm they're all implemented.
- [ ] **Architecture:** Confirm that the changes to the code are consistent with the existing architecture.
- [ ] **SOUP/OTS Software:** Any new dependencies have appropriate licenses and are documented or pinned in requirements files.
- [ ] **Unit Test:** Atomic unit tests have been added, if appropriate.
- [ ] **End-to-End Tests:** End-to-end tests have been extended or updated, if appropriate.
- [ ] **Low-level Documentation:** Comments or doc strings for affected code have been updated.
- [ ] **High-Level Documentation:** The README is updated, as appropriate.
- [ ] **Change Log:** New features or bug fixes have been added to the change log in the README.

### CODE REVIEW

File: n2y/export.py [hunk 1]
```diff
@@ -58,11 +62,21 @@ def export_page(
...
+    page_properties = _page_properties(
+        page, pandoc_format, id_property, url_property, property_map,
+    )
+    if pandoc_format.startswith("docx"):
+        output_path = Path(tempfile.mkdtemp()) / "page.docx"
+        pandoc_write_or_log_errors(pandoc_ast, pandoc_format, pandoc_options, path=output_path)
+        if not output_path.exists():
+            logger.warning("An empty page will produce corrupt docx output (%r)", page.notion_url)
+        if page_properties:
+            with OpenOfficeMetadata(output_path, "a") as oof:
```
Comment (johndgiese) [inline n2y/export.py:74]: Why is all of this code necessary?

I thought the "other-pandoc-formats" branch was already at a point where pandoc would insert the page properties into the word document metadata. Thus, I would have expected that the only changes needed to accomplish https://www.notion.so/medtech-os/Make-swap-header-footer-script-get-meta-data-from-word-files-bef0b0bf025147c19800310402c64791?pvs=4 would be in the notion-qms repository.

Comment (matt-hayden) [reply n2y/export.py:74]: For the simple-database-to-files test, the only metadata generated are the database fields Name and Tags. Of those, Tags is falsy.

**Comment by matt-hayden** [comment]
Here's an example of the embedded XML metadata format:
```
<?xml version="1.0" encoding="UTF-8"?>
<Properties xmlns="http://schemas.openxmlformats.org/officeDocument/2006/custom-properties"
            xmlns:vt="http://schemas.openxmlformats.org/officeDocument/2006/docPropsVTypes">
    <property fmtid="{D5CDD505-2E9C-101B-9397-08002B2CF9AE}" pid="2" name="Example">
        <vt:lpwstr></vt:lpwstr>
    </property>
    <property fmtid="{D5CDD505-2E9C-101B-9397-08002B2CF9AE}" pid="3" name="Name">
        <vt:lpwstr>C</vt:lpwstr>
    </property>
    <property fmtid="{D5CDD505-2E9C-101B-9397-08002B2CF9AE}" pid="4" name="Tags">
        <vt:lpwstr></vt:lpwstr>
    </property>
</Properties>
```

**Comment by matt-hayden** [comment]
After the force push, replaced by https://github.com/innolitics/n2y/pull/124

**Pull request closed without merging**